README.md

```diff
@@ -119,53 +169,3 @@ The `async` approach has two advantages over the use of
      required at each generator (`<-`) in the for-comprehension.
      This reduces the size of generated code, and can avoid boxing
      of intermediate results.
-
-## Comparison with CPS plugin
-
-The existing continuations (CPS) plugin for Scala can also be used
-to provide a syntactic layer like `async`. This approach has been
-used in Akka's [Dataflow Concurrency](http://doc.akka.io/docs/akka/2.3-M1/scala/dataflow.html)
-(now deprecated in favour of this library).
-
-CPS-based rewriting of asynchronous code also produces a closure
-for each suspension. It can also lead to type errors that are
-difficult to understand.
-
-## How it works
-
- - The `async` macro analyses the block of code, looking for control
-   structures and locations of `await` calls. It then breaks the code
-   into 'chunks'. Each chunk contains a linear sequence of statements
-   that concludes with a branching decision, or with the registration
-   of a subsequent state handler as the continuation.
- - Before this analysis and transformation, the program is normalized
-   into a form amenable to this manipulation. This is called the
-   "A Normal Form" (ANF), and roughly means that:
-     - `if` and `match` constructs are only used as statements;
-       they cannot be used as an expression.
-     - calls to `await` are not allowed in compound expressions.
- - Identify vals, vars and defs that are accessed from multiple
-   states. These will be lifted out to fields in the state machine
-   object.
- - Synthesize a class that holds:
-   - an integer representing the current state ID.
-   - the lifted definitions.
-   - an `apply(value: Try[Any]): Unit` method that will be
-     called on completion of each future. The behavior of
-     this method is determined by the current state. It records
-     the downcast result of the future in a field, and calls the
-     `resume()` method.
-   - the `resume(): Unit` method that switches on the current state
-     and runs the users code for one 'chunk', and either:
-       a) registers the state machine as the handler for the next future
-       b) completes the result Promise of the `async` block, if at the terminal state.
-   - an `apply(): Unit` method that starts the computation.
-
```

Comment:
Is the idea that this removed section will now  live somewhere else

Reply:
https://github.com/scala/scala-async/pull/237

Reply:
It is now: https://github.com/scala/scala-async/issues/242